src/Toast/_Toast.scss

```diff
@@ -5,7 +5,6 @@
   background-color: $toast-background-color;
   box-shadow: $toast-box-shadow;
   margin: 0;
-  min-width: $toast-max-width;
```

Comment:
So this `min-width` on the screen size of less than 768px was resulting in Toast to fall beyond the visible screen (towards right) and the default `.toast-container` class's `right: 0.625rem;` had no effect. 

In my opinion, we can move this inside media query with the rule `@media only screen and (min-width: 768px)` so it may work as-is for screen sizes larger than 768px. And we could override `.toast-container` class inside `@media only screen and (max-width: 768px)` and add `right: 1.25rem;`. It will work after removing this min-width property. I have also tested it and it works fine

Reply:
Yes, updating the right property [here](https://github.com/openedx/paragon/blob/master/src/Toast/_ToastContainer.scss#L14) will centralize the toast, thank you!